src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -770,8 +770,6 @@ def load_arguments(self, _):
         c.argument('enable_vtpm', enable_vtpm_type)
         c.argument('os_disk_delete_option', arg_type=get_enum_type(self.get_models('DiskDeleteOptionTypes')), min_api='2022-03-01', arg_group='Storage', help='Specify whether OS disk should be deleted or detached upon VMSS Flex deletion (This feature is only for VMSS with flexible orchestration mode).')
         c.argument('data_disk_delete_option', arg_type=get_enum_type(self.get_models('DiskDeleteOptionTypes')), min_api='2022-03-01', arg_group='Storage', help='Specify whether data disk should be deleted or detached upon VMSS Flex deletion (This feature is only for VMSS with flexible orchestration mode)')
-        c.argument('max_surge', arg_type=get_three_state_flag(), min_api='2022-11-01', is_preview=True,
-                   help='Specify it to create new virtual machines to upgrade the scale set, rather than updating the existing virtual machines.')
```

Comment:
>  is_preview=True,

Why delete the label of `is_preview`, because it has already been GA?

Reply:
It's still in preview status, https://learn.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-configure-rolling-upgrades?tabs=portal1%2Cportal2%2Cportal3%2Ccli4#concepts
Fixed